build/yamls/flow-aggregator.yml

```diff
@@ -165,12 +165,15 @@ data:
     # is not available), a value will be randomly generated, which may vary across restarts of the flow
     # aggregator.
     #observationDomainID:
+
+    # Enable sending JSON record to to the configured flow collector instead of IPFIX message.
+    #sendJSONRecord: false
```

Comment:
this should be a `recordFormat` parameter IMO, with a default value of `IPFIX` and `JSON` being another valid value

Reply:
Agreed. It will be more useful and also extendible to something else.

Reply:
Updated. Thanks!